sotodlib/io/load_smurf.py

```diff
@@ -1810,139 +2063,172 @@ def load_file(filename, channels=None, samples=None, ignore_missing=True,
+        flist = [(f, 0, None) for f in filenames]
     else:
         sample_start, sample_stop = samples
-        
+
         if archive is None and obsfiledb is None:
             outs = _get_sample_info(filenames)
         else:
             X = [archive if archive is not None else obsfiledb][0]
-            outs = [ X.lookup_file( file ) for file in filenames ]
-        stop = sample_stop 
+            outs = [X.lookup_file(file) for file in filenames]
+        stop = sample_stop
         for filename, out in zip(filenames, outs):
-            file_start, file_stop = out['sample_range']
+            file_start, file_stop = out["sample_range"]
             if stop is not None:
                 if file_start > sample_stop:
                     continue
-                stop = sample_stop-file_start
-            
-            start = max(0, sample_start-file_start)
-            flist.append( (filename, start, stop) )
-            
+                stop = sample_stop - file_start
+
+            start = max(0, sample_start - file_start)
+            flist.append((filename, start, stop))
+
     subreq = [
-        io_load.FieldGroup('data', ch_info.rchannel, timestamp_field='time'),
+        io_load.FieldGroup("data", ch_info.rchannel, timestamp_field="time"),
     ]
     if load_primary:
-        subreq.extend( [io_load.FieldGroup('primary', [io_load.Field('*', wildcard=True)])] )
+        subreq.extend(
+            [io_load.FieldGroup("primary", [io_load.Field("*", wildcard=True)])]
+        )
     if load_biases:
-        subreq.extend( [io_load.FieldGroup('tes_biases', [io_load.Field('*', wildcard=True)]),])
+        subreq.extend(
+            [
+                io_load.FieldGroup("tes_biases", [io_load.Field("*", wildcard=True)]),
+            ]
+        )
 
-    request = io_load.FieldGroup('root', subreq)
+    request = io_load.FieldGroup("root", subreq)
     streams = None
     try:
-        for filename, start, stop in tqdm( flist , total=len(flist), disable=(not show_pb)):
-            streams = io_load.unpack_frames(filename, request, streams=streams, samples=(start,stop))
+        for filename, start, stop in tqdm(
+            flist, total=len(flist), disable=(not show_pb)
+        ):
+            streams = io_load.unpack_frames(
+                filename, request, streams=streams, samples=(start, stop)
+            )
     except KeyError:
-        logger.error("Frames do not contain expected fields. Did Channel Mask change during the file?")
+        logger.error(
+            "Frames do not contain expected fields. Did Channel Mask change during the file?"
+        )
         raise
-        
-    count = sum(map(len,streams['time']))
 
-    ## Build AxisManager
+    count = sum(map(len, streams["time"]))
+
+    # Build AxisManager
     aman = core.AxisManager(
         ch_info[det_axis].copy(),
-        core.OffsetAxis('samps', count, sample_start)
+        core.OffsetAxis("samps", count, sample_start),
+        status.aman,
```

Comment:
I think instead we want to use wrap to add the status aman as one piece. So after `aman = AxisManager(..)` do something like `aman.wrap("status", status.aman)`. The way this is added right now puts a lot of lower level status things "top level" in the axismanager and I'd rather we not do that.